### PR TITLE
Add different way to define relations on models

### PR DESCRIPTION
**Pull request by stefanmajoor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,16 +271,16 @@ class AnimalStore extends Store {
 class Animal extends Model {
     @observable id = null;
     @observable name = '';
+    
+    @observable breed = this.relation(Bread);
+    @observable relatives = this.relation(AnimalStore)
```
Comment (abzainuddin) [inline README.md:276]: Add missing trailing `;`.

File: src/Model.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ import {
 import Store from './Store';
 import { invariant, snakeToCamel, camelToSnake, relationsToNestedKeys, forNestedRelations } from './utils';
 import Axios from 'axios';
+import {Relation} from "./Relations";
```
Comment (abzainuddin) [inline src/Model.js:29]: Fix styling.

File: src/__tests__/NewRelation.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    test('Can have a relation to itself', () => {
+        const animal = new Animal(null, {
+            relations: ['father'],
+        });
+
+        expect(animal.father).toBeInstanceOf(Animal);
+    })
+
+
```
Comment (abzainuddin) [inline src/__tests__/NewRelation.js:65]: Fix styling.

File: src/__tests__/NewRelation.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+})
+
+
```
Comment (abzainuddin) [inline src/__tests__/NewRelation.js:68]: Fix styling.

File: src/__tests__/NewRelation.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    });
+
+    test('Initialize one-level relation', () => {
+        const animal = new Animal(null, {
+            relations: ['kind'],
+        });
+
+        expect(animal.kind).toBeInstanceOf(Kind);
+    });
+    test('Initialize multiple relations', () => {
```
Comment (abzainuddin) [inline src/__tests__/NewRelation.js:40]: Fix styling.

File: src/__tests__/NewRelation.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
+import { Kind, Person, PersonStore} from "./fixtures/Animal";
+import {observable} from "mobx";
```
Comment (abzainuddin) [inline src/__tests__/NewRelation.js:2]: Fix styling.

File: src/__tests__/NewRelation.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+import Model from "../Model";
```
Comment (abzainuddin) [inline src/__tests__/NewRelation.js:3]: Fix styling.

File: src/__tests__/NewRelation.js [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+import {BinderApi} from "../index";
```
Comment (abzainuddin) [inline src/__tests__/NewRelation.js:4]: Fix styling.

**Review by abzainuddin — CHANGES_REQUESTED** [review]
Please add a test for nested relations defined in the new way.

**Comment by stefanmajoor** [comment]
> Please add a test for nested relations defined in the new way.

It is there

**Review by abzainuddin — APPROVED** [review]

**Pull request merged**